packages/diagrams/frontend/sirius-components-diagrams/src/renderer/helper-lines/useHelperLines.tsx

```diff
@@ -125,44 +128,240 @@ const getHelperLines = (
   return noHelperLines;
 };
 
+const getHelperLinesForResize = (
+  change: NodeDimensionChange,
+  resizingNode: Node<NodeData>,
+  nodes: Node<NodeData>[]
+): HelperLines => {
+  const noHelperLines: HelperLines = { horizontal: null, vertical: null, snapX: 0, snapY: 0 };
+  if (resizingNode.positionAbsolute && change.dimensions) {
+    const getNodeById = (id: string | undefined) => nodes.find((n) => n.id === id);
```

Comment:
you could declare this function outside since you're using it in getHelperLinesForMove too

Reply:
I don(t think I could extract this method, I need to provide the `nodes` ...